src/shared/modules/itemEditor/itemEditorDuck.js

```diff
@@ -15,6 +16,14 @@ export const setSelectedItem = item => {
   }
 }
 
+export const fetchData = (id, entityType) => {
+  return {
+    type: FETCH_DATA_ON_SELECT,
+    id,
+    entityType
```

Comment:
entityType is undefined.
pass the entity type as it is being used in Epic.

Reply:
done with changes